### DISCUSSION TITLE
Landing Page Design Mockup

### DISCUSSION THREAD

**Discussion by conradKE** [StanleyMasinde/events254Web-archive#107] [Ideas]
hello what do you guys think about [this ](https://app.moqups.com/drJzZuWcfm/view/page/ad64222d5)new mockup design
![8Hs0J](https://user-images.githubusercontent.com/22999104/127159380-bf53a6e7-8710-43d7-b615-caf8d5326918.jpg)

**Comment by StanleyMasinde** [comment]
It looks great. I'll steal some ideas from it

**Comment by conradKE** [comment]
wabee try and see

**Comment by StanleyMasinde** [comment]
@azim254, I'll put this into consideration. 

## Landing Page
Here is what we have upcoming. I am starting with a mobile first approach. I can fix the desktop later
### iPhone 12
![events254 test_(iPhone 12 Pro)](https://user-images.githubusercontent.com/35426405/168054832-0d11c0cb-517a-4f3e-9328-af2a4b68c2c6.png)

### iPad Pro
![events254 test_(iPad Pro)](https://user-images.githubusercontent.com/35426405/168055128-f8d61847-ec39-4ebe-a89b-69c2fc128c2a.png)


## Event view WIP
### iPhone 12
![events254 test_(iPhone 12 Pro) (1)](https://user-images.githubusercontent.com/35426405/168055513-54875821-52fc-4f3f-83b0-52910f8fbbb3.png)

### iPad pro
![events254 test_(iPad Pro) (1)](https://user-images.githubusercontent.com/35426405/168055648-db60e094-262c-4044-8358-da10662cb5c5.png)


Please share any feedback you

**Reply by conradKE** [reply]
nice
